x/mint/keeper/hooks.go

```diff
@@ -21,20 +21,20 @@ func (k Keeper) AfterEpochEnd(ctx sdk.Context, epochIdentifier string, epochNumb
 		if epochNumber < params.MintingRewardsDistributionStartEpoch {
 			return
 		} else if epochNumber == params.MintingRewardsDistributionStartEpoch {
-			k.SetLastHalvenEpochNum(ctx, epochNumber)
+			k.SetLastReductionEpochNum(ctx, epochNumber)
 		}
 		// fetch stored minter & params
 		minter := k.GetMinter(ctx)
 		params := k.GetParams(ctx)
 
 		// Check if we have hit an epoch where we update the inflation parameter.
-		// Since epochs only update based on BFT time data, it is safe to store the "halvening period time"
```

Comment:
Does this comment line even make sense, I don't see us time anywhere

Reply:
ah maybe we should reword as "we measure time between reductions in number of epochs. This avoids issues with measuring in block numbers, as epochs have fixed intervals, with very low variance at the relevant sizes"